### PR TITLE
CIT-833: Remove CIOT

### PR DESCRIPTION
**Pull request by areyeslo**
- Remove front end

### CODE REVIEW

File: cit3.0-web/src/components/Page/PowerBi/CriteriaSearch.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Config from "../../../Config";
 import "./PowerBi.css";
 
 export default function CriteriaSearch() {
-  const [report, setReport] = useState();
+  const [, setReport] = useState();
```
Comment (bashbang) [inline cit3.0-web/src/components/Page/PowerBi/CriteriaSearch.js:10]: is this correct syntax to have the , in there?

File: cit3.0-web/src/components/Page/PowerBi/Compare.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import Config from "../../../Config";
 import "./PowerBi.css";
 
 export default function Compare() {
-  const [report, setReport] = useState();
+  const [, setReport] = useState();
```
Comment (bashbang) [inline cit3.0-web/src/components/Page/PowerBi/Compare.js:10]: is this correct syntax to have the comma in the set?

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default function PowerBi() {
   const user = useSelector((state) => state.user);
   const configuration = useConfiguration();
   const [currentPage, setCurrentPage] = useState(null);
-  const [currentPageData, setCurrentPageData] = useState(null);
+  const [, setCurrentPageData] = useState(null);
```
Comment (bashbang) [inline cit3.0-web/src/components/Page/PowerBi/PowerBi.js:27]: same....is this correct with the comma?

**Review by bashbang — CHANGES_REQUESTED** [review]

File: cit3.0-web/src/components/Page/PowerBi/CriteriaSearch.js (hunk 1 above)
Comment (areyeslo) [reply cit3.0-web/src/components/Page/PowerBi/CriteriaSearch.js:10]: if we just need to setReport, you can skip items while destructuring by using a comma without an variable or constant name.

File: cit3.0-web/src/components/Page/PowerBi/Compare.js (hunk 2 above)
Comment (areyeslo) [reply cit3.0-web/src/components/Page/PowerBi/Compare.js:10]: It is that correct and remove warnings during compilation.

**Review by bashbang — APPROVED** [review]

**Pull request merged**